storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/db/entities/Queen.java

```diff
@@ -0,0 +1,163 @@
+package com.pushtorefresh.storio.sample.db.entities;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.util.Log;
+
+import com.pushtorefresh.storio.sample.db.tables.AntsTable;
+import com.pushtorefresh.storio.sample.db.tables.QueensTable;
+import com.pushtorefresh.storio.sqlite.annotations.StorIOSQLiteColumn;
+import com.pushtorefresh.storio.sqlite.annotations.StorIOSQLiteType;
+import com.pushtorefresh.storio.sqlite.operations.put.PutResults;
+import com.pushtorefresh.storio.sqlite.queries.Query;
+import com.pushtorefresh.storio.sqlite.StorIOSQLite;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import rx.Observer;
+import rx.Subscription;
+import rx.android.schedulers.AndroidSchedulers;
+
+import static com.pushtorefresh.storio.sample.ui.Toasts.safeShowShortToast;
+import static rx.android.schedulers.AndroidSchedulers.mainThread;
+
+@StorIOSQLiteType(table = QueensTable.TABLE)
+public class Queen {
```

Comment:
Yes of course! :-D
That's only another example - made only for me. Don't look at it. ;-)